src/VertexManager.sol

```diff
@@ -409,7 +409,9 @@ contract VertexManager is Initializable, UUPSUpgradeable, OwnableUpgradeable, Re
     /// @notice Returns the price on Vertex of a given by product.
     /// @param id The ID of the product to get the price of.
     function getPrice(uint32 id) public view returns (uint256) {
-        return IClearinghouse(endpoint.clearinghouse()).getOraclePriceX18(id);
+        // If id is 0 (quote), return default price.
+        if (id == 0) return 1e18;
+        return endpoint.getPriceX18(id);
```

Comment:
don't you need to add this to the endpoint interface for this as well?

Reply:
it's added here https://github.com/ElixirProtocol/vertex-contracts/blob/e130cfc174c6c0d7e788561e611f2c8ea3cce491/src/interfaces/IEndpoint.sol#L80 if that's what you are referring to

Reply:
ah i missed it

Reply:
fixed price ack.